### ISSUE TITLE
YAML::Mo Undefined subroutine &main::main:: Line 5

### ISSUE THREAD

**Issue by Oswulf** [ingydotnet/yaml-pm#159]
Attempting to install a module via CPAN I get the following error:

CPAN: YAML loaded ok (v1.16)
Alert: While trying to 'dump' YAML file
 '/root/.cpan/build/RSS-Parser-Lite-0.12-1.yml'
with 'YAML' the following error was encountered:
  Undefined subroutine &main::main:: called at /usr/local/share/perl/5.20.2/YAML/Mo.pm line 5.

This would appear to be the reemergence of an old issue:

https://rt.cpan.org/Public/Bug/Display.html?id=74487

**Comment by btisdall** [comment]
This is also happening for me:
```
# ... test and install completes, then
  /usr/bin/make install  -- OK
Alert: While trying to 'dump' YAML file
 '/tmp/.cpan/build/YAML-1.16-0.yml'
with 'YAML' the following error was encountered:
  Undefined subroutine &main::main:: called at /opt/perl-5.20.2/lib/site_perl/5.20.2/YAML/Mo.pm line 5.
```

**Comment by xsawyerx** [comment]
Can we please revert this release? It also broke Dancer (1 and 2). I assume quite a few other things.

**Comment by perlpunk** [comment]
Fixed in YAML 1.17, thanks
